app/assets/javascripts/moss.js

```diff
@@ -0,0 +1,32 @@
+function toggleOptions(dropdown, table) {
+  const $dropdown = $(dropdown);
+  const $table = $(table);
+
+  $table.toggle();
+  if ($table.is(':hidden')) {
+    $dropdown.children('.expand-more').show();
+    $dropdown.children('.expand-less').hide();
+  } else {
+    $dropdown.children('.expand-more').hide();
+    $dropdown.children('.expand-less').show();
+  }
+}
+
+function filterCourses(name) {
+  $(".filterableCourse").each(function(i, e) {
```

Comment:
right now it seems like the courses need to match all specified keywords, shouldn't it be one of the keywords if for example the assessment names were different?

Reply:
Changed so that
- show if any keywords match
- now able to search on semester too